lib/inspec/utils/telemetry/data_series.rb

```diff
@@ -0,0 +1,45 @@
+require 'json'
+
+# A minimal Dataseries Object
+# Stores the name of the data series and an array of data.
+# Stored data should be a object that supports #to_s
+module Inspec::Telemetry
+  class DataSeries
+    def initialize(name)
+      @name = name
+      @enabled = true
```

Comment:
If `@data` is intrinsic to the data series, it should be initialized in the constructor